include/aikido/planner/ompl/Planner.hpp

```diff
@@ -296,17 +298,23 @@ trajectory::InterpolatedPtr planOMPL(
 /// to a state on original path
 /// \param _originalTraj The untimed trajectory obtained from the planner, 
 /// needs simplifying.
-trajectory::InterpolatedPtr simplifyOMPL(statespace::StateSpacePtr _stateSpace,
-                                           statespace::InterpolatorPtr _interpolator,
-                                           distance::DistanceMetricPtr _dmetric,
-                                           constraint::SampleablePtr _sampler,
-                                           constraint::TestablePtr _validityConstraint,
-                                           constraint::TestablePtr _boundsConstraint,
-                                           constraint::ProjectablePtr _boundsProjector,
-                                           double _maxDistanceBtwValidityChecks,
-                                           double _timeout, size_t _maxEmptySteps,
-                                           double _rangeRatio, double _snapToVertex,
-                                           trajectory::InterpolatedPtr _originalTraj);
+std::pair <std::unique_ptr<trajectory::Interpolated>, bool> simplifyOMPL(statespace::StateSpacePtr _stateSpace,
+                                                        statespace::InterpolatorPtr _interpolator,
+                                                        distance::DistanceMetricPtr _dmetric,
+                                                        constraint::SampleablePtr _sampler,
+                                                        constraint::TestablePtr _validityConstraint,
+                                                        constraint::TestablePtr _boundsConstraint,
+                                                        constraint::ProjectablePtr _boundsProjector,
+                                                        double _maxDistanceBtwValidityChecks,
+                                                        double _timeout, size_t _maxEmptySteps,
+                                                        trajectory::InterpolatedPtr _originalTraj);
+
+
+namespace{
+    std::unique_ptr<trajectory::Interpolated> aikido2ompl(::ompl::geometric::PathGeometric *path, 
+                                                    statespace::StateSpacePtr _stateSpace,
+                                                    statespace::InterpolatorPtr _interpolator);
+} // namespace
```

Comment:
- Anonymous `namespace`s should [almost](http://stackoverflow.com/a/358823) never be used in header files. This is a fine function that we could promote to a new `planning/ompl/Trajectory.hpp` file.
- Rename this function to match our other conversion functions, e.g. added in #147. I suggest`toInterpolatedTrajectory` for this function and `toOmplPathGeometric` for the inverse. @jslee02 Objections?
- Take the `PathGeometric` argument by `const &` instead of non-`const` pointer, unless there is a strong reason to do otherwise.